source/docs/software/wpilib-tools/axon/retraining-mobilenet.rst

```diff
@@ -0,0 +1,108 @@
+:orphan:
+
+Retraining Mobilenet V2
+=======================
+
+Upload a dataset
+----------------
+
+To retrain the mobilenet object detection model using Axon, first upload the dataset you would like to retrain with, using the button on the tree to the left.
+
+.. image:: images/axon-mobilenet-1.png
+
+Once you have uploaded a dataset, you should see it in the tree above this button.
+
+Create a project
+----------------
+
+Next, you must create a project. A project will hold the data from the retraining, and any tflite exports or tests you create with the retrained model.
+
+To create a project, click the add project button, below the add dataset button.
+
+.. image:: images/axon-mobilenet-2.png
+
+A dialog will appear, asking you to name the project. Pick a name and click create. When you create your project, you will see it above the button to the left.
+
+Click on the project, and you will be taken to its page.
+
+Start retraining
+----------------
+
+At the top of the project window, there are several parameters to specify how you want the model to be retrained.
+
+Epochs: The number of times the dataset will be passed through the model during training.
```

Comment:
Bolden the terms

Reply:
Resolved in f98414b793038d44e85bd196885ef80c97775627